### ISSUE TITLE
Non-orthogonal slices of data

### ISSUE THREAD

**Issue by d-v-b** [napari/napari#320]
## 🚀 Feature
Render arbitrary planes through volumetric data. 

## Motivation
With isotropic volumetric data, there's no guarantee that the underlying data aligns with the axes along which the data are represented. [Bigdataviewer](https://github.com/bigdataviewer) and tools derived from it allow rendering arbitrary slices of volumetric data, which allows interactively re-aligning data to get informative views on underlying features that might not be aligned the array axes.  This is pretty nice. 

N.B. I have no idea how rotating the slicing plane would work with spatial dimensions > 3. That might demand too much.

**Comment by sofroniewn** [comment]
This would be pretty cool, and something we should think about as we move towards our support for rapidly switching which dimensions are shown (XY, YZ, XZ, etc.) and 3D rendering in general. Thanks for flagging

**Comment by brisvag** [comment]
This was made possible by #3023 and followups! Closing.

**Comment by sofroniewn** [comment]
One comment just so folks know is that this only works on data small enough to fit on the GPU. Hopefully one day we'll extend to big data too!

**Comment by jni** [comment]
Yeah to be honest I would consider reopening this given that it's quite unlikely #3023 is actually usable by @d-v-b. @perlman and I do see a future where the slicing logic is abstract and async enough for BDV-style slicing to work in napari.

Or do we want to open a new issue "BIG non-orthogonal slices of data"? :joy:

**Comment by brisvag** [comment]
Right, I wasn't thinking of larger-than-memory data! I guess I'll reopen :)

**Comment by liu-ziyang** [comment]
Hi there, I would like to close this issue to consolidate our recent effort on the big data set slicing implementation plan. Thanks! (if you would like to know more about this plan, we are aggregating it here as a starting point: https://github.com/orgs/napari/projects/16)

**Comment by jni** [comment]
I'll reopen this because we don't have an issue tracking larger-than-RAM non-orthogonal reslicing (here or in the linked project), so this is not resolved or duplicating any other issue.
